frontend/src/scenes/popup/CreatePopupPage.jsx

```
@@ -1,140 +1,320 @@
+          setUrl={setUrl}
+        />
+      )}
+      leftAppearance={() => (
+        <PopupAppearance
+          data={stateList}
+          setPopupSize={setPopupSize}
+          popupSize={popupSize}
+        />
+      )}
+    />
+  );
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)